Docs grammar and typos check (#6064)
vuestorefront/vue-storefront: packages/core/docs/guide/user-profile.md

## Loading current user

> To access the data of the currently logged-in user, we can use another property of `useUser` called simply `user`.

Reviewer: ```suggestion
To access the data of the currently logged-in user, you can use the `user` property of `useUser` composable.
```
